.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: "CI"
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  ci-merge:
+    runs-on: ubuntu-latest
```

Comment:
Nit: Is it a leftover? Should it be bullseye?

Reply:
No it is not a leftover. The available options for github runners are macos, winows and ubuntu ([available runners](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on)).
So we use an ubuntu machine but run our things inside a container that runs the bullseye image.